scripts/release/test_all

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#
+# Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+readonly SCRIPTS_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# run the normal test_all script
+("$SCRIPTS_DIR"/../test_all)
+all_tests_ok=$?
+
+# extra tests for the release
+echo checking pod push authorization ...
+if ! pod trunk me | grep -q 'MaterialComponents'; then
+  all_tests_ok=0
+  echo
+  echo "Failed! Register an account and session: https://guides.cocoapods.org/making/getting-setup-with-trunk.html"
+  echo "Continuing with next test..."
+  echo
+fi
+
+echo lint podspec ...
```

Comment:
Broke up into two as suggested. One ended up just being a subcommand so I put that back into the releaseing.md